beacon_node/network/src/sync/manager.rs

```diff
@@ -883,20 +922,30 @@ impl<T: BeaconChainTypes> SyncManager<T> {
         seen_timestamp: Duration,
     ) {
         match request_id {
-            RequestId::SingleBlock { id } => self.block_lookups.single_block_lookup_response(
-                id,
-                peer_id,
-                block,
-                seen_timestamp,
-                &mut self.network,
-            ),
-            RequestId::ParentLookup { id } => self.block_lookups.parent_lookup_response(
-                id,
-                peer_id,
-                block,
-                seen_timestamp,
-                &mut self.network,
-            ),
+            RequestId::SingleBlock { id } => self
+                .block_lookups
+                .single_lookup_response::<BlockRequestState<Current>>(
+                    id,
+                    peer_id,
+                    block,
+                    seen_timestamp,
+                    &self.network,
+                ),
+            RequestId::SingleBlob { .. } => {
+                crit!(self.log, "Blob received during block request"; "peer_id" => %peer_id  );
```

Comment:
If I'm understanding this correctly, I think this should be: 

```suggestion
                crit!(self.log, "Block received during blob request"; "peer_id" => %peer_id  );
```